ow_sim_tests/scripts/test_arm_consistency.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python
+
+# The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+# Research and Simulation can be found in README.md in the root directory of
+# this repository.
+
+import rospy
+import os
+import actionlib
+import pandas as pd
+import tf2_ros
+import roslib
+import unittest
+import owl_msgs.msg
+import numpy as np
+import random
+import matplotlib.pyplot as plt
+import rospkg
+from mpl_toolkits.mplot3d import Axes3D
```

Comment:
Although the code is not used directly, but in order to make the 3d plot, we must import the Axes3D